kotsadm/web/src/components/troubleshoot/GenerateSupportBundle.jsx

```diff
@@ -88,15 +89,20 @@ class GenerateSupportBundle extends React.Component {
       })
         .then(async (res) => {
           const response = await res.json();
-          this.setState({
-            supportBundles: response.supportBundles,
-            loadingSupportBundles: false,
-          });
+          if(!res.ok) {
+            this.setState({ loadingSupportBundles: false, errorMsg: `Unable to get list of bundles: ${response.error}` });
+          }
+          if(res.ok) {
```

Comment:
how about adding a `return` to the `!res.ok` instead of adding another check here? just to make this cleaner